README.md

```diff
@@ -53,4 +53,24 @@ make install  # install the CRDs to the cluster
 ```bash
 kustomize build config/crd > crds.yaml  # crds
 kustomize build config/default > manifests.yaml  # all other manifests
+```
+
+## Configuration
+
+A file path with following parameters is required to be passed to the operator with `--config <path>` flag.
+
+```yaml
+clusterName: my-cluster
+storage:
+  defaultSize: 4Gi
+aws:
+  accountID: 123456789
+  region: eu-west-2
+  auth:
+    accessKey: access-key
+    secretKey: secret-key
```

Comment:
no, originally i thought id have to provide direct access credentials when running locally, but it just seemed to work without them through the OIDC. I can remove them for now since its misleading at the moment.